src/controllers/developers.js

```diff
@@ -62,3 +58,29 @@ export const postDeveloper = async (req, res, next) => {
     return next(err);
   }
 };
+
+export const getDeveloper = async (req, res, next) => {
+  const connection = createDbConnection();
+  const Developer = connection.model('Developer', developerSchema);
+  const { id } = req.params;
+  try {
+    const developer = await Developer.find({ _id: id });
+
+    if (!developer.length) {
+      throw new Error("Developer doesn't exist");
+    }
+
+    // developer = developer.toJSON();
```

Comment:
why is this line commented?

Reply:
It throws this error:
`
{
    "error": "_developer2.toJSON is not a function"
}
`